### PR TITLE
Add yadm

### PR DESCRIPTION
**Pull request by TheLocehiliosan**
## Checklist for submitting a pull request to `Terminals Are Sexy`:

- [X] I have carefully **read and comply** with the [Contributing Guidelines](https://github.com/k4m4/terminals-are-sexy/blob/master/CONTRIBUTING.md) of this repo.
- [X] I have **searched** the [PRs](https://github.com/k4m4/terminals-are-sexy/pulls) of this repo and believe that this is not a duplicate.

 

- **Title**: yadm - Yet Another Dotfiles Manager
- **Link**: https://thelocehiliosan.github.io/yadm/
- **Category**: Tools and Plugins
- **Description**: Seamless git-based management of your dotfiles.

yadm is a dotfile management tool with 3 main features: Manages files across systems using a single Git repository. Provides a way to use alternate files on a specific OS or host. Supplies a method of encrypting confidential data so it can safely be stored in your repository.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [homebrew-cask](https://github.com/caskroom/homebrew-cask) - CLI workflow for the administration of macOS applications distributed as binaries.
 * [mps-youtube](https://github.com/mps-youtube/mps-youtube) - Terminal based YouTube player and downloader
 * [colorls](https://github.com/athityakumar/colorls) - Beautify the terminal's `ls` command, with color and font-awesome icons.
+* [yadm](https://thelocehiliosan.github.io/yadm/) - Seamless git-based management of your dotfiles.
```
Comment (k4m4) [inline readme.md]: Please change the URL to the GitHub repo address (i.e. [github.com/TheLocehiliosan/yadm](https://github.com/TheLocehiliosan/yadm))

**Comment by TheLocehiliosan** [comment]
I've updated the URL to be the official yadm.io address.

**Comment by k4m4** [comment]
Thanks! Could you please also squash your commits into one?

**Comment by TheLocehiliosan** [comment]
Sure, I've also rebased this on top of master.

**Comment by k4m4** [comment]
I just noticed that you changed the link to [yadm.io](yadm.io) instead of [github.com/TheLocehiliosan/yadm](https://github.com/TheLocehiliosan/yadm). Could you please change it to the GitHub repo page? Thanks.

**Comment by TheLocehiliosan** [comment]
Yes, I used yadm.io because that's the official website for yadm. In the future, the source code might switch to a different site, but the official site will always be yadm.io.

**Pull request merged**